File: demo/src/main/java/com/example/demo/model/Transaction.java
package com.example.demo.model;

import lombok.Getter;
import lombok.NoArgsConstructor;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;
import java.util.Date;

@Entity
@Table(name= "TRANSACTION")
@Getter
@NoArgsConstructor
public class Transaction {

    @Id
    @Column(columnDefinition = "transaction_id")
    private Long transactionId;

    @Column(columnDefinition = "card_no")
    private String cardNo;
    @Column(columnDefinition = "sender")
    private String sender;
    @Column(columnDefinition = "receiver")
    private String receiver;
    @Column(columnDefinition = "amount")
    private double amount;
    @Column(columnDefinition = "card_no")
    private Date transactionDate;

}

File: ARBoost/Model/Transaction.swift
//
//  Transaction.swift
//  ARBoost
//
//  Created by Mert Gökçen on 18.08.2021.
//

import Foundation

class Transaction {
    var id:Int
    var cardNo:String
    var sender:String
    var reciever:String
    var amount:Double
    var type:String
    var date:Date?
    
    init(id:Int,cardNo:String,sender:String,reciever:String,
         amount:Double,type:String,date:Date){
        self.id = id
        self.cardNo = cardNo
        self.sender = sender
        self.reciever = reciever
        self.amount = amount
        self.type = type
        self.date = date
    }
}
